xlive/Blam/Engine/main/main_game.cpp

```diff
@@ -51,7 +51,11 @@ void main_game_launch_set_coop_player_count(int32 player_count)
 {
     if (!IN_RANGE_INCLUSIVE(player_count, 1, k_number_of_users))
     {
+#ifdef MSVC
         error(2, __FUNCTION__": invalid player count %d (must be from 1-%d)", player_count, k_number_of_users);
+#else
+        error(2, "%s: invalid player count %d (must be from 1-%d)", __FUNCTION__, player_count, k_number_of_users);
```

Comment:
I don't see why this wouldn't work using MSVC

Reply:
Original string appends the function name to the string while this isn't possible on non-msvc compilers.